xds/internal/client/client_watchers.go

```diff
@@ -234,12 +222,12 @@ func (c *Client) watchLDS(serviceName string, cb func(ListenerUpdate, error)) (c
 	return c.watch(wi)
 }
 
-// watchRDS starts a listener watcher for the service..
+// WatchRoute starts a listener watcher for the service..
 //
 // Note that during race (e.g. an xDS response is received while the user is
 // calling cancel()), there's a small window where the callback can be called
 // after the watcher is canceled. The caller needs to handle this case.
-func (c *Client) watchRDS(routeName string, cb func(RouteConfigUpdate, error)) (cancel func()) {
+func (c *Client) WatchRoute(routeName string, cb func(RouteConfigUpdate, error)) (cancel func()) {
```

Comment:
Maybe `WatchRouteConfig` or `WatchRouteConfiguration`? because we have another first-class `Route` type even though we can't setup a watch on that, it might be better to be explicit.

Reply:
Done.